lte/gateway/c/core/BUILD.bazel

```diff
@@ -26,1117 +26,1168 @@ cc_asn1_library(
-        "//lte/gateway/c/core/common:backtrace",
-        "//lte/gateway/c/core/common:common_defs",
-        "//lte/gateway/c/core/common:dynamic_memory_check",
-        "//lte/gateway/c/core/oai/common/glogwrapper:glog_logging",
-        "//lte/gateway/c/core/oai/lib/bstr:bstrlib",
-        "//lte/gateway/c/core/oai/lib/directoryd:directoryd_client",
-        "//lte/gateway/c/core/oai/lib/event_client:eventd_client",
-        "//lte/gateway/c/core/oai/lib/hashtable",
-        "//lte/protos:ha_orc8r_cpp_grpc",
-        "//lte/protos:ha_service_cpp_grpc",
-        "//lte/protos:mconfigs_cpp_proto",
-        "//lte/protos:mobilityd_cpp_grpc",
-        "//lte/protos:pipelined_cpp_grpc",
-        "//lte/protos:s1ap_service_cpp_grpc",
-        "//lte/protos:s6a_service_cpp_grpc",
-        "//lte/protos:s6a_service_cpp_proto",
-        "//lte/protos:sctpd_cpp_grpc",
-        "//lte/protos:session_manager_cpp_grpc",
-        "//lte/protos:sms_orc8r_cpp_grpc",
-        "//lte/protos:spgw_service_cpp_grpc",
-        "//lte/protos:spgw_service_cpp_proto",
-        "//lte/protos:subscriberauth_cpp_grpc",
-        "//lte/protos:subscriberdb_cpp_grpc",
-        "//lte/protos/oai:common_types_cpp_proto",
-        "//lte/protos/oai:mme_nas_state_cpp_proto",
-        "//lte/protos/oai:ngap_state_cpp_proto",
-        "//lte/protos/oai:s1ap_state_cpp_proto",
-        "//lte/protos/oai:sgw_state_cpp_proto",
-        "//orc8r/gateway/c/common/async_grpc:async_grpc_receiver",
-        "//orc8r/gateway/c/common/config:mconfig_loader",
-        "//orc8r/gateway/c/common/ebpf",
-        "//orc8r/gateway/c/common/logging",
-        "//orc8r/gateway/c/common/sentry:sentry_wrapper",
-        "//orc8r/gateway/c/common/service303",
-        "//orc8r/protos:eventd_cpp_proto",
-        "//orc8r/protos:mconfigs_cpp_proto",
-        "//orc8r/protos:redis_cpp_proto",
-        "@cpp_redis",
-        "@libfluid_base//:fluid_base",
-        "@libfluid_msg//:fluid_msg",
-        "@liblfds//:lfds710",
-        "@system_libraries//:czmq",
-        "@system_libraries//:libconfig",
-        "@system_libraries//:libfd",
-        "@system_libraries//:libgnutls",
-        "@system_libraries//:libnettle",
-        "@system_libraries//:sctp",
-    ],
+    deps = MME_DEPS,
 )
 
-cc_binary(
-    name = "oai_mme",
-    srcs = [
-        "oai/oai_mme/oai_mme.c",
-    ],
+# TODO(#11892): Giant Bazel label is bad practice, break this up
+cc_library(
+    name = "lib_mme_oai",
+    srcs = MME_SRCS + GTPV2_C_C + S11_C + UDP_C,
+    hdrs = MME_HDRS,
     copts = [
-        "-DEMBEDDED_SGW",  # TODO(#11638): Feature flags fixups
+        "-DCMAKE_BUILD_TYPE=\\\"CMAKE_BUILD_TYPE\\\"",  # TODO(#11897): needs alternative approach or deprecation
+        "-DEMBEDDED_SGW=0",
+        "-DS6A_OVER_GRPC=0",
     ],
+    deps = MME_DEPS,
+)
+
+cc_binary(
+    name = "agw_of",
+    srcs = ["oai/oai_mme/oai_mme.c"],
+    copts = ["-DEMBEDDED_SGW=1"],
     linkstatic = False,
```

Comment:
Regarding both MME binaries: Can MME be linked statically now that we wrapped glog?

Reply:
Yes, this will be a separate PR.